tap_tiktok_ads/streams.py

```diff
@@ -163,7 +165,9 @@ def transform_advertisers_records(records, bookmark_value):
 
 def get_bookmark_value(stream_name, bookmark_data, advertiser_id):
```

Comment:
Do add function comment to explain the if condition.

Reply:
Added comment